crates/pico_macros/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "pico_macros"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+
+[dependencies]
+pico_core = { path = "../pico_core" }
+proc-macro2 = "1.0"
+syn = "2.0.87"
+quote = "1.0"
+darling = "0.20.10"
```

Comment:
can we try to do `workplace = true` for all dependencies of non-iso non-relay crates?